components/Common/Search/States/WithSearchResult.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Result } from '@orama/orama';
+import NextLink from 'next/link';
+import type { FC } from 'react';
+
+import type { SearchDoc } from '@/components/Common/Search/States/WithSearchBox';
+import { pathToBreadcrumbs } from '@/components/Common/Search/utils';
+import { highlighter } from '@/next.orama.mjs';
+
+import styles from './index.module.css';
+
+type SearchResultProps = {
+  hit: Result<SearchDoc>;
+  searchTerm: string;
+};
+
+export const WithSearchResult: FC<SearchResultProps> = props => {
+  const isAPIResult = props.hit.document.siteSection.toLowerCase() === 'api';
+  const basePath = isAPIResult ? 'https://nodejs.org/docs/latest' : '/en';
```

Comment:
Please remove this basePath thing for `/en`; I'd say only the APIResults need a "basePath". If you use our Link component it already gets handled gracefully (the localization bit)